tests/utils/test_uri.py

```diff
@@ -338,33 +342,69 @@ def test_is_databricks_acled_artifacts_uri():
     )
 
 
+def _get_databricks_profile_uri_test_cases():
```

Comment:
This helper function is basically just parametrizing the existing test cases in `test_get_databricks_profile_uri_from_artifact_uri` by `databricks` vs `databricks-uc` URI scheme